graphite/client.go

```diff
@@ -68,7 +79,55 @@ func NewClient(carbon string, carbon_transport string, write_timeout time.Durati
 	}
 }
 
-func pathFromMetric(m model.Metric, prefix string) string {
+func loadContext(tmpl_context map[string]interface{}, m model.Metric) map[string]interface{} {
```

Comment:
Can you move all that to rules.go ? Then we just call rules.pathsFromMetric() in this file.